api-reference/10 UI Components/Errors and Warnings/W1021.md

```diff
@@ -3,7 +3,7 @@ id: ErrorsUIWidgets.W1021
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs in the [Popup](/Documentation/ApiReference/UI_Components/dxPopup/), [Popover](/Documentation/ApiReference/UI_Components/dxPopover/), [Tooltip](/Documentation/ApiReference/UI_Components/dxTooltip/), and [LoadPanel](/Documentation/ApiReference/UI_Components/dxLoadPanel/) if the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property is set to a DOM element that does not exist.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Change the [container](/Documentation/ApiReference/UI_Components/dxPopup/Configuration/#container) property value to an existing element.
```

Comment:
"existing element" is to strict for me. It can be a selector of a dom element. It's also enough to remove container property

Reply:
Noted